pyhomematic/devicetypes/sensors.py

```diff
@@ -190,6 +190,39 @@ class MotionV2(Motion, HelperSabotage):
     """Motion detection version 2."""
     pass
 
+class MotionIP(HMBinarySensor, HMSensor):
```

Comment:
HelperLowBat will not work (tried it before). datapoints.py shows "LOW_BAT" instead of "LOWBAT" in HelperLowBat. Got a way to work around this?